.github/workflows/array-api.yml

```diff
@@ -0,0 +1,82 @@
+name: Array API
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8"]
+
+    steps:
+      - name: Checkout Dask
+        uses: actions/checkout@v2
+        with:
+          path: dask
+      - name: Checkout Array API tests
+        uses: actions/checkout@v2
+        with:
+          repository: data-apis/array-api-tests
+          path: array-api-tests
+          ref: 2022.02.24
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          pushd array-api-tests
+          pip install -e '../dask[array]'
+          pip install -r requirements.txt
+      - name: Run Array API tests
+        env:
+          ARRAY_API_TESTS_MODULE: dask.array_api
+        run: |
+          pushd array-api-tests
+
+          # Skip testing functions with known issues
+          cat << EOF >> skips.txt
```

Comment:
Does this need to be passed to `pytest` somehow?

Reply:
The array-api-tests repo automatically uses skips.txt to skip tests https://github.com/data-apis/array-api-tests/blob/master/conftest.py#L84. Ideally you could just pass these XFAILS on the command line, but that doesn't seem to be possible (https://stackoverflow.com/questions/68764967/xfail-pytest-tests-from-the-command-line)